src/ReviewService.NativePrompters/ReviewService.NativePrompters.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<LangVersion>10.0</LangVersion>
+		<LangVersion>12.0</LangVersion>
 		<Nullable>enable</Nullable>
-		<TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-windows10.0.19041</TargetFrameworks>
+		<TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-windows10.0.19041</TargetFrameworks>
```

Comment:
I think this should be a major update because it's a binary breaking change. People can no longer use this on their .NET 6 projects.